AirshipKit/AirshipKit/ios/UAChannelCapture.m

```diff
@@ -74,10 +74,15 @@ - (void)checkClipboard {
     if (!self.push.channelID) {
         return;
     }
-    
-    NSDate *date = [self.dataStore objectForKey:UAChannelCaptureEnabledKey];
-    
-    if ([self.push backgroundPushNotificationsEnabled] && [date compare:[NSDate date]] == NSOrderedAscending) {
+
+    if ([self.push backgroundPushNotificationsEnabled]) {
+        NSDate *enabledUntilDate = [self.dataStore objectForKey:UAChannelCaptureEnabledKey];
+        if (!enabledUntilDate || [enabledUntilDate compare:[NSDate date]] == NSOrderedAscending) {
```

Comment:
Correct, and so if it's nil the condition will never evaluate to true. That means if channel capture was never explicitly enabled by setting a date, it will still run, even when it shouldn't.

From the changelog:

Behavior Changes
----------------
- Channel Capture tool is now disabled by default if the app is able
  to receive background push. A new action has been added to enable the
  tool for a limited duration.